bindata/network/ovn-kubernetes/self-hosted/ovnkube-master.yaml

```diff
@@ -382,7 +383,11 @@ spec:
                 echo "NB DB Raft leader is unknown to the cluster node."
                 exit 1
               else
-                /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null
+                # enable memory triming once
+                if [ ! -f /var/run/ovn/ovnnb_db_mem_trim.txt ]; then
+                  /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null
+                  echo "done" > /var/run/ovn/ovnnb_db_mem_trim.txt
```

Comment:
I think the convention is to use `touch something-executed`

e.g.
https://github.com/openshift/machine-config-operator/blob/master/templates/common/_base/files/configure-ovs-network.yaml#L9

```shell    
    # This file is not needed anymore in 4.7+, but when rolling back to 4.6
    # the ovs pod needs it to know ovs is running on the host.
    touch /var/run/ovs-config-executed
```